perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/VosManagerMethod.java

```diff
@@ -373,7 +373,8 @@ public Void call(ApiCaller ac, Deserializer parms)
 					role, parms.readBoolean("onlyDirectAdmins"));
 			} else {
 				return ac.getVosManager().getAdmins(ac.getSession(),
-					ac.getVoById(parms.readInt("vo")));
+					ac.getVoById(parms.readInt("vo")),
+					null,parms.readBoolean("onlyDirectAdmins"));
```

Comment:
This change is not correct. Role can't be null for this method. Please return back the state before your change there. It was deprecated but functional.